osmosis-1/osmosis.zone_assets.json

```diff
@@ -1904,6 +1904,19 @@
       "osmosis_main": false,
       "osmosis_frontier": true,
       "osmosis_info": true
+    },
+    {
+      "chain_name": "arbitrum",
+      "base_denom": "0x912CE59144191C1204E64559FE8253a0e49E6548",
+      "pools": {
+        "OSMO": 1011
+      },
+      "osmosis_main": false,
+      "osmosis_frontier": true,
+      "osmosis_info": true,
+      "override_properties": {
```

Comment:
Override symbol isn't required, because the symbol is already ARB.

Reply:
Thank you. Edited the file to remove the override symbol